third_party/wasm-polyfill/jslib/load-wasm.js

```diff
@@ -0,0 +1,192 @@
+// This file implements a browser utility function to asychronously fetch,
+// decode and compile a given WebAssembly module. The caller passes in the url
+// of the .wasm file and the returned promise resolves to a compiled (but
+// unlinked) module.
+
+var loadWebAssembly = (function() {
+  // *** Environment setup code ***
+  var ENVIRONMENT_IS_WEB = typeof window === 'object';
```

Comment:
Hmm, I wonder why we need to duplicate all of this here? It is a bit unfortunate to have to maintain these in multiple places?